contrib/account-command/gpg-json/README.md

```diff
@@ -0,0 +1,38 @@
+# GPG-based JSON database
+
+## Prerequisites
+
+- [gnupg](https://gnupg.org)
+- a private key, which we're assume belongs to me@example.com
+- [jq](https://stedolan.github.io/jq/)
+
+## Setup
+
+### Generate your credentials database
+
+Create a `credentials.json` JSON file containing your credentials.
+The structure is an array of objects.
+Each object represents an account.
+
+```json
+[
+  { "host": "imap.gmail.com",    "user": "me@gmail.com",    "pass": "s3cr3t" },
+  { "host": "imap.fastmail.com", "user": "me@fastmail.com", "pass": "foob4r" },
+  { "host": "smtp.fastmail.com", "user": "me@fastmail.com", "pass": "foob4r" }
+]
+```
+
+### Encrypt the credentials database
+
+Encrypt your credentials file using gpg and get rid of the original file.
+
+```sh
+$ gpg -r me@example.com -e credentials.json && rm credentials.json
+```
+
+This will generate a `credentials.json.gpg` encrypted file.
+
+### Setup your account command
+
+Place the `credentials.sh` shell script alongside your `credentials.json.gpg` file.
```

Comment:
Hmm...
I'm not sure that we should encourage users to **copy** the script.
If we ever updated/improved the script, they wouldn't benefit.

Reply:
Is `contrib`:
- a suggestion: Here's how you _could_ do it
- an example: Use this working script

Reply:
Could we add some sample config:
(everyone like to cut'n'paste, if it needs tweaking afterwards)

```
set account_command = "/usr/share/doc/neomutt/account-command/gpg-json/credentials.sh"
```